DiceWorker/build.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
```

Comment:
The individual worker build scripts will currently fail if you didn't run `download_dependencies.sh` beforehand. Can you make the scripts run this? (It should be a no-op in case the dependencies already exist)

Reply:
done.